code/datums/recipe.dm

```diff
@@ -95,7 +95,7 @@
 					needed_items -= itype
 			else
 				needed_items -= itype
```

Comment:
The break is necessary to avoid unnecessarily using stuff up in container_contents.
```suggestion
					needed_items -= itype
					break
			else
				needed_items -= itype
				break
```